js/signup.js

```diff
@@ -1,115 +1,167 @@
-  e.key === ENTER_KEY && $jsPasswordInput.focus();
+$emailInput.addEventListener('keydown', (e) => {
+  if (e.key === KEY.ENTER) {
+    $passwordInput.focus();
+  }
 });
 
-$jsPasswordInput.addEventListener('keydown', (e) => {
-  e.key === ENTER_KEY && $jsPasswordCheckInput.focus();
+$passwordInput.addEventListener('keydown', (e) => {
+  if (e.key === KEY.ENTER) {
+    $passwordConfirmInput.focus();
+  }
 });
 
-$jsPasswordCheckInput.addEventListener('keydown', (e) => {
-  e.key === ENTER_KEY && $joinButton.click();
+$passwordConfirmInput.addEventListener('keydown', (e) => {
+  if (e.key === KEY.ENTER) {
+    $joinButton.click();
+  }
 });
 
 const $btnPasswordIcon = $('.btn-password-icon');
+const $eyeOpenIcon = $('.eye-open-icon');
+const $eyeCloseIcon = $('.eye-close-icon');
 
 $btnPasswordIcon.addEventListener('click', () => {
-  if ($jsPasswordInput.type === 'text') {
-    hidePassword($jsPasswordInput);
-    return;
-  }
-  if ($jsPasswordInput.type === 'password') {
-    showPassword($jsPasswordInput);
-  }
+  displayPassword(
+    $passwordInput.type,
+    $passwordInput,
+    $eyeOpenIcon,
+    $eyeCloseIcon
+  );
 });
 
-const $btnPasswordCheckIcon = $('.btn-password-check-icon');
+const $btnPasswordConfirmIcon = $('.btn-password-confirm-icon');
 
-$btnPasswordCheckIcon.addEventListener('click', () => {
-  if ($jsPasswordCheckInput.type === 'text') {
-    hidePassword($jsPasswordCheckInput);
-    return;
-  }
-  if ($jsPasswordCheckInput.type === 'password') {
-    showPassword($jsPasswordCheckInput);
-  }
+$btnPasswordConfirmIcon.addEventListener('click', () => {
+  displayPassword(
+    $passwordConfirmInput.type,
+    $passwordConfirmInput,
+    $eyeOpenIcon,
+    $eyeCloseIcon
+  );
 });
 
 $joinButton.addEventListener('click', () => {
+  //클래스형태로 수정하기
   const errorStyle = '1px solid var(--red)';
-  const emailStyle = $jsEmailInput.style.border;
-  const passwordStyle = $jsPasswordInput.style.border;
-  const passwordCheckStyle = $jsPasswordCheckInput.style.border;
+  const emailStyle = $emailInput.style.border;
+  const passwordStyle = $passwordInput.style.border;
+  const passwordConfirmStyle = $passwordConfirmInput.style.border;
 
   if (
     emailStyle === errorStyle ||
     passwordStyle === errorStyle ||
-    passwordCheckStyle === errorStyle
+    passwordConfirmStyle === errorStyle
   ) {
     return;
   }
 
-  location.href = './folder.html';
+  signup($emailInput.value, $passwordInput.value);
 });
+
+const signup = async (email, password) => {
+  console.log(`email = ${email} // password = ${password}`);
+  const bodyData = {
+    email,
+    password,
+  };
+  try {
+    const response = await POST_API(API_URL.AUTH.SIGN_UP, loginData);
+    const result = JSON.parse(await response.text());
+
+    if (response.status === 200) {
+      localStorage.setItem('accessToken', result.data.accessToken);
+      location.href = './folder.html';
+    }
+  } catch (error) {
+    console.log(error);
```

Comment:
log보다는 error를 사용해주는게 좋을 것 같아요!
```suggestion
    console.error(error);
```